modules/caddyhttp/reverseproxy/httptransport.go

```diff
@@ -171,12 +171,25 @@ func (HTTPTransport) CaddyModule() caddy.ModuleInfo {
 	}
 }
 
+var (
+	allowedVersions       = []string{"1.1", "2", "h2c", "3"}
+	allowedVersionsString = strings.Join(allowedVersions, ", ")
```

Comment:
Nit: This could just be inlined below with the error message, since it won't be happening THAT often, in fact quite rarely; but, I'm okay with it either way. So you don't have to change it.